source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,346 @@
+
+
+## Internal Escelation
+
+
+**Simon can you help fill in this**
+
+
+Inform the following channels of the situation and put forward a contactable member of staff from our team.
+
+
+`#ask-modernisation-platform`
+
+
+`#modernisation-platform`
+
+
+`#modernisation-platform-alerts`
+
+
+
+## Disaster Recovery Types
+
+
+**Let’s work the problem, people. Let’s not make things worse by guessing.**
+
+
+- London has been lost but **no services need to migrate**. [Click here]().
+- London has been lost, and **all services need to migrate**. [Click here]().
+- Loss of a **single account**. [Click here]().
+
+
+## London has been lost but returning soon.
+
+
+### Getting a decision
+
+
+Once a ticket has been raised with AWS, and we have engaged with them and have total confirmation that the infrastructure will be returning within X number of minutes or hours, we will need to pass this information on for an informed decision to be made above.
+
+
+Only when we have this, can we update users on what is next.
+
+
+### Priority list
+
+
+If there is limited availability, which is a possibility, these are the priority accounts that need to be checked before we can focus on member-accounts.
+
+
+
+|Account|Information|
+|----------------- |------------------------------------------------ |
+|Core-Shared-Services|Creates the core shared services account resources. |
+| Core-Network-Services | Creates the core networking account resources; this is the highest priority. |
+|Core-Logging|Creates the core logging account resources |
+|Core-Security|Creates the core security account resources |
+|Cloud-Platform-Transit-Gateways|Across all regions. |
+|Core-VPC-*|Creates the core VPC resources in the VPC accounts, needed after the core-networking-services. |
+| Modernisation-Platform | Creates key resources such as S3 state buckets for the platform |
+
+
+### Test Accounts
+
+
+If there is limited availability, power off all infrastructure in the test accounts. All are useless until the platform is running.
+
+
+
+## Single Account
+
+
+Dont forget the statefile
+
+
+### Will it be returning
+
+
+If this is a test account, waiting may be the easier idea. If AWS can promise us this is a temporary issue with an estimated fix time, we can pass this to the team and agree our next step.
+
+
+### Rebuild account
+
+
+Assuming this is a production account with no time given to return, please follow the steps.
+
+
+#### Workflows
+
+
+Rather than rebuilding an account through workflows, I would run manually, however, technically, you could run the account creation process workflows.
+
+
+#### Manually
+
+
+1.) Run a terraform plan against the production workspace, in modernisation-platform .
+2.) All changes should appear, using your admin credentials, run an apply.
```

Comment:
I don't think this is the best idea. I think that GitHub Actions should still be the method we use for deploying. It allows visibility, and access to logs for review.

Reply:
Agree, and it's also impossible, unless you have a root account access.